helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -36,6 +44,74 @@ spec:
+    # record when pods of a daemonset with label "label_giantswarm_io_monitoring_basic_sli" are down
+    - expr: |
+        label_replace(kube_daemonset_status_number_unavailable
+        and on(daemonset,cluster_id,cluster_type,namespace, workload_name)
+        kube_daemonset_labels{label_giantswarm_io_monitoring_basic_sli='true'}, "service", "$1", "workload_name", "(.*)" )
+      labels:
+        class: MEDIUM
+        area: managed-apps
+      record: raw_slo_errors
+    # record when pods of a deployment with label "label_giantswarm_io_monitoring_basic_sli" are down
+    - expr: |
+        label_replace(kube_deployment_status_replicas_unavailable
+        and on(deployment,cluster_id,cluster_type,namespace, workload_name) 
+        kube_deployment_labels{label_giantswarm_io_monitoring_basic_sli='true'}, "service", "$1", "workload_name", "(.*)" )
+      labels:
+        class: MEDIUM
+        area: managed-apps
+      record: raw_slo_errors
+    # record when pods of a statefulset with label "label_giantswarm_io_monitoring_basic_sli" are down
+    - expr: |
+        label_replace(kube_statefulset_status_replicas - kube_statefulset_status_replicas_current
+        and on(statefulset,cluster_id,cluster_type,namespace, workload_name) 
+        kube_statefulset_labels{label_giantswarm_io_monitoring_basic_sli='true'}, "service", "$1", "workload_name", "(.*)" )
+      labels:
+        class: MEDIUM
+        area: managed-apps
+      record: raw_slo_errors
+    - expr: |
+        label_replace(kube_daemonset_status_desired_number_scheduled
+        and on(daemonset,cluster_id,cluster_type,namespace, workload_name)
+        kube_daemonset_labels{label_giantswarm_io_monitoring_basic_sli='true'}, "service", "$1", "workload_name", "(.*)" )
+      labels:
+        class: MEDIUM
+        area: managed-apps
+      record: raw_slo_requests
+    - expr: |
+        label_replace(kube_deployment_status_replicas
+        and on(deployment,cluster_id,cluster_type,namespace, workload_name) 
+        kube_deployment_labels{label_giantswarm_io_monitoring_basic_sli='true'}, "service", "$1", "workload_name", "(.*)" )
+      labels:
+        class: MEDIUM
+        area: managed-apps
+      record: raw_slo_requests
+    - expr: |
+        label_replace(kube_statefulset_status_replicas
+        and on(statefulset,cluster_id,cluster_type,namespace, workload_name) 
+        kube_statefulset_labels{label_giantswarm_io_monitoring_basic_sli='true'}, "service", "$1", "workload_name", "(.*)" )
+      labels:
+        class: MEDIUM
+        area: managed-apps
+      record: raw_slo_requests
+      # -- 99.5% availability
+    - expr: sum by (service) (raw_slo_errors{area="managed-apps"} - raw_slo_errors{area="managed-apps"}) + 1-0.995
+      record: slo_target
+
+      # -- generic stuff
+      # -- standard burnrates based on https://sre.google/workbook/alerting-on-slos/#6-multiwindow-multi-burn-rate-alerts
+    - expr: "vector(36)"
+      record: slo_burnrate_high
```

Comment:
We merged this already but should this not be named slo_burnate_fast and slow instead of high and low as this is how fast we burn our slos?

Reply:
I understand "high rate" like we are burning fast, so to me it sounds similar.

Reply:
I can accept this :D I'm just checking :)